loader/loader_windows.c

```diff
@@ -51,6 +51,8 @@
 #include <dxgi1_6.h>
 #include "adapters.h"
 
+#include <appmodel.h>
```

Comment:
Is this header guaranteed to always be present on Windows 10? Just making sure there isn't some build requirement that is being added. This probably only affects some of the non-standard windows based builds, like MinGW.

Reply:
The two APIs I'm retrieving out of this header were available in the Windows 8.1 SDK:
* https://docs.microsoft.com/en-us/windows/win32/api/appmodel/nf-appmodel-getpackagepathbyfullname
* https://docs.microsoft.com/en-us/windows/win32/api/appmodel/nf-appmodel-getpackagesbypackagefamily

That said... I don't see them in the MinGW header repository. I guess this probably needs some CMake logic to detect the header and turn off this code if it's not present.

Reply:
I see a `__MINGW32__` macro guard in loader_windows.c, but I'm not well versed in mingw to know if thats sufficient, nor correct. I'm tempted to not do anything about it now since it isn't one of the first party supported platforms. Then anyone who wants to update the loader for mingw can fix it 'properly', instead of a hack-dash attempt at one that may require later fixing anyhow.

Reply:
Works for me. I'll resolve this comment then.